pom.xml

```diff
@@ -0,0 +1,102 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>gov.acwi</groupId>
+    <artifactId>wqp-etl-nwis-lookups</artifactId>
+    <version>0.1.0</version>
+
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.1.3.RELEASE</version>
+    </parent>
+
+    <properties>
+        <java.version>1.8</java.version>
+        <cida.maven.url>http://internal.cida.usgs.gov/maven</cida.maven.url>
+    </properties>
+    
+    <scm>
+        <connection>scm:git:https://github.com/NWQMC/etl-nwis-lookups.git</connection>
+        <developerConnection>scm:git:git@github.com:NWQMC/etl-nwis-lookups.git</developerConnection>
+        <url>https://github.com/NWQMC/etl-nwis-lookups.git</url>
+        <tag>HEAD</tag>
+    </scm>
+    <repositories>
+        <repository>
+            <id>nexus</id>
+            <name>CIDA Nexus</name>
+            <url>${cida.maven.url}/content/groups/public/</url>
+        </repository>
+    </repositories>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-batch</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.postgresql</groupId>
+            <artifactId>postgresql</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.oracle</groupId>
+            <artifactId>ojdbc6</artifactId>
+            <version>11.2.0.2.0-1</version>
+            <scope>provided</scope>
```

Comment:
Maybe take the scope off as we will need it included in the jar (will not be deployed to a tomcat containing it...)